frontend/controller/actions/identity.js

```diff
@@ -21,6 +22,22 @@ import type { Key } from '../../../shared/domains/chelonia/crypto.js'
 import { handleFetchResult } from '../utils/misc.js'
 import { encryptedAction } from './utils.js'
 
+const requireToSuccess = async (fnToSuccess) => {
+  if (typeof fnToSuccess !== 'function') {
+    return
+  }
+
+  let failureCount = 0
+  do {
+    try {
+      return await fnToSuccess()
+    } catch (err) {
+      console.error('[requireToSuccess:]', err)
+      failureCount++
+    }
+  } while (failureCount < 10)
+}
+
```

Comment:
@Silver-IT This function can now be removed once #2027 is merged into this PR. Could you please have a look at that PR and merge it when ready?

See my comment here too: https://github.com/okTurtles/group-income/pull/2027#pullrequestreview-2088739741